kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/job/service/impl/AbstractPipelineJobAPIImpl.java

```diff
@@ -89,8 +90,8 @@ public Optional<String> start(final PipelineJobConfiguration jobConfig) {
         GovernanceRepositoryAPI repositoryAPI = PipelineAPIFactory.getGovernanceRepositoryAPI(PipelineJobIdUtils.parseContextKey(jobId));
         String jobConfigKey = PipelineMetaDataNode.getJobConfigPath(jobId);
         if (repositoryAPI.isExisted(jobConfigKey)) {
-            log.warn("jobId already exists in registry center, ignore, jobConfigKey={}", jobConfigKey);
-            return Optional.of(jobId);
+            log.error("jobId already exists in registry center, jobConfigKey={}", jobConfigKey);
+            throw new PipelineJobAlreadyExistsException(jobId);
```

Comment:
It's better for idempotence if not throw exception

Reply:
`PipelineJobIdAlreadyExistsException` to represent `jobId` existing might be better

Reply:
Ok, Considering the idempotence, I close it now.